src/coreclr/pal/src/thread/process.cpp

```diff
@@ -2664,18 +2664,22 @@ FlushProcessWriteBuffers()
         int status = pthread_mutex_lock(&flushProcessWriteBuffersMutex);
         FATAL_ASSERT(status == 0, "Failed to lock the flushProcessWriteBuffersMutex lock");
 
+#ifndef TARGET_BROWSER
```

Comment:
This is same code as what's in src/coreclr/gc/unix/gcenv.unix.cpp. It should be ifdefed same way.

(Ideally, we would move this to minipal to avoid the duplication - but that would be larger change.)

Reply:
https://github.com/dotnet/runtime/issues/117977